test/models/vote_test.rb

```diff
@@ -0,0 +1,25 @@
+require "test_helper"
+
+describe 'Vote' do
```

Comment:
I didn't know this about tests, but the tests that had errors seemed to be fixed when I changed this line to `describe Vote do` (remove the quotes)